docs/development_roles.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Roles in development"
+---
+
+During development of Home Assistant integrations, you will come across different roles that are involved in the process. This document explains these roles and their responsibilities.
+
+## Core members
+
+Core members are contributors with write access to the Home Assistant Core repository.
+They have the ability to review and merge pull requests, manage issues, and maintain the overall quality of the codebase. 
+Core members are responsible for ensuring that contributions adhere to the project's guidelines and standards.
+
+## Integration owners
+
+Formerly known as "codeowners", integration owners are core members who have taken on the responsibility of maintaining specific integrations within Home Assistant.
+Comments and concerns raised by integration owners are held in high regard, as they possess in-depth knowledge of the integration they oversee.
+Integration owners are mentioned in every pull request that changes the code of their integration, and in every issue that is opened for their integration.
+
+## Core team
```

Comment:
We need to change the higher level names and clearly distinguish the OHF part in the role names.

The HA GitHub org is currently already reflecting that.

That said; I do want to change the naming fully for this one, and consider forming sn architectural group of some sort.